jwql/utils/utils.py

```diff
@@ -458,6 +479,59 @@ def filename_parser(filename):
         r"_(?P<instrument>(nircam|niriss|nirspec|miri|fgs))"\
         r"_(?P<optical_elements>((?!_)[\w-])+)"
 
+    # Stage 3 WFSS filenames
+    # e.g. jw01076-o124_s000001220_nircam_f322w2-grismr_x1d.fits cal.fits c1d.fits
+    #This should not be used anymore...
```

Comment:
As in, these files are no longer produced by the pipeline?

Reply:
Exactly. They will be present in MAST until the new file naming scheme is in place and reprocessing is done and, I'm assuming, the MAST folks will have to delete them manually. But, once the initial reprocessing is done with the new file naming scheme, all the info will be available in the new files and we won't need to think about the old ones.

Reply:
Cool, we will keep the comment when this day eventually comes.